scripts/devops_tasks/find_change_log.ps1

```diff
@@ -0,0 +1,94 @@
+﻿
+param (
+  $workingDir,
+  $version
+)
+$RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?))"
+
+
+function ExtractReleaseNotes($changeLogLocation)
+{
+  $releaseNotes = @{}
+  $contentArrays = @{}
+  if ($changeLogLocation.Length -eq 0)
+  {
+    return $releaseNotes
+  }
+
+  try {
+    $contents = Get-Content $changeLogLocation
+
+    # walk the document, finding where the version specifiers are and creating lists
+    $version = ""
+    foreach($line in $contents){
+      if ($line -match $RELEASE_TITLE_REGEX)
+      {
+        $version = $matches["version"]
+        $contentArrays[$version] = @()
+      }
+
+      $contentArrays[$version] += $line
+    }
+
+    # resolve each of discovered version specifier string arrays into real content
+    foreach($key in $contentArrays.Keys)
+    {
+      $releaseNotes[$key] = New-Object PSObject -Property @{
+        ReleaseVersion = $key
+        ReleaseContent = $contentArrays[$key] -join [Environment]::NewLine
+      }
+    }
+  }
+  catch
+  {
+    Write-Host "Error parsing $changeLogLocation."
+    Write-Host $_.Exception.Message
+  }
+
+  return $releaseNotes
+}
+
+
+function VerifyPackages($rootDirectory)
+{ 
+   #This function verifies version in history.md for a given package
+   try
+   {
+      $historyFiles = Get-ChildItem -Path $rootDirectory -Recurse -Include "history.md"
+      if ($historyFiles -eq $null)
+      { 
+         exit(0)
+      }
+
+      #Find current version of package from _version.py and package name from setup.py
+      $changeFile = @($historyFiles)[0]
+      #Get Version and release notes in each change log files
+      $releaseNotes = ExtractReleaseNotes -changeLogLocation $changeFile
+      if ($releaseNotes.Count -gt 0)
```

Comment:
I would like to output the contents of the changelog that we see. Not only that, but we expect exactly _one_ release note for the given version. If that version exists, dump the content that is associated with it to stdout. This will give confidence to people verifying their stuff in CI.

Reply:
Thanks for the feedback. I have pushed updated changes as per your comments.